src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapeGame.java

```diff
@@ -136,7 +136,16 @@ public int forceStart() {
 		startOngoingPhase();
 		return 0;
 	}
-	
+
+	/**
+	* @return      0 if successful<br> 
+	* 				-1 if ongoing phase has not started
+	*/
```

Comment:
incoherent javadocs. When returning only one value don't even need to have javadocs. Remove it and return void.